src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -124,24 +173,77 @@ export default class ReplyField extends React.Component<Props, State> {
         setCursorPosition(editorState.getSelection().getFocusOffset());
     };
 
+    stopDefaultEvent = (event: React.SyntheticEvent): void => {
+        event.preventDefault();
+        event.stopPropagation();
+    };
+
+    setPopupListActiveItem = (index: number): void => this.setState({ activeItemIndex: index });
+
+    handleReturn = (event: React.KeyboardEvent): DraftHandleValue => {
+        const { activeItemIndex, popupReference } = this.state;
+
+        if (!popupReference) {
+            return 'not-handled';
+        }
+
+        this.stopDefaultEvent(event);
+        this.handleSelect(activeItemIndex);
+
+        return 'handled';
+    };
+
+    handleArrow = (event: React.KeyboardEvent): void => {
+        const { activeItemIndex, editorState, popupReference } = this.state;
+        const collaborators = this.getCollaborators(getActiveMentionForEditorState(editorState));
```

Comment:
I think `getCollaborators` can be updated to call `getActiveMentionForEditorState(editorState)` itself instead of needing to pass it in each time.